src/lava/magma/core/model/py/ports.py

```diff
@@ -48,7 +48,11 @@ def peek(self):
         pass
 
     def probe(self) -> bool:
-        pass
+        return ft.reduce(
+            lambda acc, csp_port: acc and csp_port.probe(),
+            self._csp_recv_ports,
+            True,
+        )
```

Comment:
I don't understand what this does. Please add a descriptive comment.

Reply:
Added a descriptive comment in latest commit.